snap/gadget.go

```diff
@@ -52,25 +61,34 @@ type GadgetVolume struct {
 // type when we actually handle these.
 
 type VolumeStructure struct {
-	Label       string          `yaml:"filesystem-label"`
-	Offset      string          `yaml:"offset"`
-	OffsetWrite string          `yaml:"offset-write"`
-	Size        string          `yaml:"size"`
-	Type        string          `yaml:"type"`
-	ID          string          `yaml:"id"`
-	Filesystem  string          `yaml:"filesystem"`
-	Content     []VolumeContent `yaml:"content"`
-	Update      VolumeUpdate    `yaml:"update"`
+	Name        string               `yaml:"name"`
```

Comment:
What is the difference between the label and the name?

Reply:
Label is just a filesystem label. Name written as partition name if GPT schema is in use, otherwise it's informative.

Reply:
Ah, can you document that in the structure please?